libraries/c_sdk/standard/mqtt/src/iot_mqtt_subscription_container.c

```diff
@@ -51,7 +51,7 @@
  * otherwise.
  */
 static bool _packetMatch( _mqttSubscription_t * pSubscription,
-                          void * pMatch );
+                          const void * pMatch );
```

Comment:
why to remote const qualifier?

Reply:
I have added the const qualifier as the matching params should not change.